migrator/src/main/java/uk/nhs/digital/ps/migrator/model/hippo/Attachment.java

```diff
@@ -88,8 +89,13 @@ public void download() {
                     log.debug("Downloaded {} Mb / {} Mb", getMb(bytesRead), getMb(bytesExpected));
                 }
             }
-        } catch (IOException e) {
-            throw new RuntimeException("Error occurred downloading attachment", e);
+            return true;
+        } catch (Exception e) {
+            file.delete(); // Attempt to delete the file as it may be partially downloaded
```

Comment:
Probably unlikely to happen, but could the partially downloaded file be in use/locked by a process, and an exception thrown by the delete attempt?

Reply:
Good point, If this happens the migration will just fail so hopefully it is unlikely enough that a total failure is ok as we can resolve the issue and run again.